h/scripts/init_elasticsearch.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+"""
+Initialize the search index.
+
+Usage:
+
+    python3 -m h.scripts.init_elasticsearch conf/development.ini
+
+"""
+import argparse
+import os
+
+from pyramid.paster import bootstrap
+
+from h import search
+
+
+def main():
+    parser = argparse.ArgumentParser(description="Initialize the search index.")
+
+    parser.add_argument(
+        "config_uri",
+        help="the URI to the config file, e.g. 'conf/production.ini'",
+    )
+
+    args = parser.parse_args()
+
+    # In production environments a short Elasticsearch request timeout is
+    # typically set and initializing a new search index can take longer than
+    # the timeout, so override any custom timeout with a high value.
+    os.environ["ELASTICSEARCH_CLIENT_TIMEOUT"] = "30"
+
+    with bootstrap(args.config_uri) as env:
```

Comment:
This is what a script can do if it needs Pyramid settings or a request object. This script shouldn't really need to do this, it should be completely Pyramid-independent like `init_db.py` is, but `search.get_client()` currently takes a Pyramid settings dict as argument so we'd need to do some refactoring to decouple search index initialization from Pyramid.